mkdocs.yml

```
@@ -17,6 +17,7 @@ nav:
       - Configuration: "setup/configuration.md"
       - Customizing: "setup/customizing.md"
       - Custom cards: "setup/custom_cards.md"
+      - Adaptive Dashboard: "setup/adaptive_dash.md"
```

Also add custom_actions to it
```suggestion
      - Adaptive Dashboard: "setup/adaptive_dash.md"
      - Custom Actions: "setup/custom_actions.md"
```

---

Thank you I forgot it !